src/containers/donation/index.css

```diff
@@ -25,20 +25,54 @@
   font-weight: 400;
 }
 
-.header {
-  padding-top: 1.8rem;
+@media only screen and (max-width: 900px) {
+  .introParagraph,
+  .donateParagraph,
+  .helpParagraph {
+    text-align: justify;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    font-weight: 400;
+  }
+}
+
+.donationHeader {
   margin-bottom: 7rem;
+  width: 100%;
 }
 
-.title {
-  font-size: 3rem;
+@media only screen and (max-width: 900px) {
```

Comment:
```@media only screen and (max-width: 900px) ``` for this breakpoint you can write it only one time and put everything you want to style inside it, as I can see you have five or six breakpoints but with the same number ```(max-width: 900px)```  so just write it one time and put everything you want to style inside it